===================================================================
RCS file: /cvs/gnome/gtk+/gtk/gtktreeview.c,v
retrieving revision 1.569
retrieving revision 1.570
diff -u -r1.569 -r1.570
--- gtktreeview.c	2006/09/16 04:40:50	1.569
+++ gtk+-2.10.6/gtk/gtktreeview.c	2006/09/25 14:25:33	1.570
@@ -14624,7 +14624,7 @@
   old_grid_lines = priv->grid_lines;
   priv->grid_lines = grid_lines;
   
-  if (!GTK_WIDGET_REALIZED (widget))
+  if (GTK_WIDGET_REALIZED (widget))
     {
       if (grid_lines == GTK_TREE_VIEW_GRID_LINES_NONE &&
 	  priv->grid_line_gc)
